feature_engine/scaling/mean_normalization.py

```diff
@@ -0,0 +1,186 @@
+# Authors: Vasco Schiavo <vasco.schiavo@protonmail.com>
+# License: BSD 3 clause
+
+from typing import List, Optional, Union
+
+import pandas as pd
+
+from feature_engine._base_transformers.base_numerical import BaseNumericalTransformer
+from feature_engine._check_init_parameters.check_variables import (
+    _check_variables_input_value,
+)
+from feature_engine._docstrings.fit_attributes import (
+    _feature_names_in_docstring,
+    _n_features_in_docstring,
+    _variables_attribute_docstring,
+)
+from feature_engine._docstrings.init_parameters.all_trasnformers import (
+    _variables_numerical_docstring,
+)
+from feature_engine._docstrings.methods import (
+    _fit_not_learn_docstring,
+    _fit_transform_docstring,
+    _inverse_transform_docstring,
+)
+from feature_engine._docstrings.substitute import Substitution
+
+
+@Substitution(
+    variables=_variables_numerical_docstring,
+    variables_=_variables_attribute_docstring,
+    feature_names_in_=_feature_names_in_docstring,
+    n_features_in_=_n_features_in_docstring,
+    fit=_fit_not_learn_docstring,
+    fit_transform=_fit_transform_docstring,
+    inverse_transform=_inverse_transform_docstring,
+)
+class MeanNormalizationScaler(BaseNumericalTransformer):
+    """
+    The MeanNormalizationScaler() applies the mean normalization scaling techinques
```

Comment:
Here I'd say that the meannormalizationscaler substracts the mean and then divides the result by the value range. And add a small formula of the trasformation